core/src/edu/asu/c3simulator/screens/EmploymentScreen.java

```diff
@@ -82,35 +86,57 @@ public void clicked(InputEvent event, float x, float y)
 		}
 	}
 	
-	@SuppressWarnings("unused")
+	/**
+	 * Displays a list of randomly generated employees in a window when the "New Employee"
+	 * button is selected.
+	 */
 	private class NewEmployeeButtonListener extends ClickListener
 	{
 		@Override
 		public void clicked(InputEvent event, float x, float y)
 		{
-			newEmployeeOptionsPanel.clear();
+			newEmployeeOptionsWindow.clear();
+			Table availableNewEmployees = new Table();
+			
 			// TODO: Do not keep number of employees generated magic
-			for (int employeesGenerated = 0; employeesGenerated < 3; employeesGenerated++)
+			for (int employeesGenerated = 0; employeesGenerated < 5; employeesGenerated++)
 			{
 				Employee employee = EmployeeFactory.getRandomEmployee();
-				Actor employeeLabel = addEmployee(employee, newEmployeeOptionsPanel,
-						false);
-				employeeLabel.addListener(new HireEmployeeListener(employee));
+				availableNewEmployees.add(createNewEmployeeComponent(employee)).expand()
+						.fill().space(20).row();
```

Comment:
Much improved. Nice job.

Another possible improvement is moving createNewEmployeeComponent(employee) to its own line, e.g:
Actor employeeComponent = createNewEmployeeComponent(employee);